app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -1,11 +1,13 @@
 class Admin::CustomOrganisationsController < AdminController
+  helper_method :sort_column, :sort_direction
+
   def index
-    @custom_organisation_names = CustomOrganisationName.all
+    order_custom_orgs
```

Comment:
same change as below please?

Reply:
You mean same as the create method? @cybojenix